packages/redot-parse/readme.md

```diff
@@ -19,14 +19,12 @@ npm install redot-parse
 ## Usage
 
 ```js
-var unified = require("unified");
```

Comment:
@ChristianMurphy Since this will change to ESModule, it is likely that ERR_REQUIRE_ESM exception may be raised when this code is executed.

I thought it would be better to modify the code in Useage and mention that it is not available from CommonJS format packages.

Alternatively, you could go for the Dual package format, which is more expensive but supports both ESModule and CommonJS.
In that case, however, it would be better to keep the documentation in ESModule format.

Perhaps it might be beneficial to update the Usage section of the code and make a note that it's incompatible with packages in the CommonJS format.

Alternatively, you might consider opting for a dual package format, which, while more demanding, accommodates both ESModule and CommonJS. If you go this route, it'd be best to stick with documentation in the ESModule format.

Reply:
Yep, I'm in the process of updating the documentation.
Still work in progress, but a preview of what I have for this package:

---

## Install

This package is [ESM only](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).
In Node.js (version 12.20+, 14.14+, or 16.0+), install with [npm][]:

```sh
npm install redot-parse
```

In Deno with [`esm.sh`][esmsh]:

```js
import redotParse from 'https://esm.sh/redot-parse@1'
```

In browsers with [`esm.sh`][esmsh]:

```html
<script type="module">
  import redotParse from 'https://esm.sh/redot-parse@1?bundle'
</script>
```

## Use

Say we have the following module `example.js`:

```js
import {unified} from 'unified'
import redotParse from 'redot-parse'
import redotSvg from 'redot-svg'

main()

async function main() {
  const file = await unified()
    .use(redotParse)
    .use(redotSvg)
    .process('digraph {Hello -> World}')

  console.log(String(file)) 
}
```

Reply:
I plan on publishing as ESM only (as most of `unified` does).
Dual packaging ESM and CJS has a bunch of problems around the [dual package hazard](https://nodejs.org/dist/v13.7.0/docs/api/esm.html#esm_dual_package_hazard) and buggy/inconsistent handling of dual packaging by build tools.

Reply:
Thanks!

I am relieved to hear that the documentation has been prepared.

I didn't know about the dual package hazard, so I learned a lot.